test/expect/TestScript.test_loop_unroll_unused_counter.expect

```diff
@@ -3,7 +3,7 @@ graph(%x : Dynamic) {
   %2 : int = prim::TensorToNum(%x)
   %3 : int = prim::Constant[value=1]()
   %4 : int = prim::Constant[value=8]()
-  %5 : int = aten::div(%2, %4)
+  %5 : int = prim::FloorDiv(%2, %4)
```

Comment:
@jamesr66a should this be exportable to ONNX? If so I can write a pass to change it to aten::div.

Reply:
I don't think we need to support it, unless we also add support for `//`. It's only emitted by loop unrolling, which doesn't run before ONNX.